src/index.html

```diff
@@ -11,7 +11,149 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          Name:
+          <input
+          type="text"
+          name="name"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="term"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="field">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="6"
+          maxlength="20"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            > Yes
```

Comment:
Let's not put text in the same line with other tags
```suggestion
            <input
              type="radio"
              name="loveCats"
              value="yes"
            > 
              Yes
```